services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/OpenHouseInternalRepositoryImpl.java

```diff
@@ -111,8 +114,8 @@ public TableDto save(TableDto tableDto) {
               tableIdentifier,
               writeSchema,
               partitionSpec,
-              storageManager
-                  .getDefaultStorage()
+              storageSelector
+                  .selectStorage(tableDto.getDatabaseId(), tableDto.getTableId())
```

Comment:
how are we able to ensure this change is safe for all implementations of `allocagteTableLocation` for example the implementation extended in li-openhouse

Reply:
Currently the only selector implementation we have is the DefaultStorageSelector that defaults to the cluster level storage so this change should have no impact.

There are multiple unit tests that test the behavior of allocateTableLocation which would fail if the behavior has changed